Sarah Hong/Homework/Week-01/Wednesday/Arrays & Functions /index.html

```diff
@@ -0,0 +1,11 @@
+<html>
+<head>
+  </head>
+  <title>Javascript Excercises</title>
+  <script src="js/homework_02.js"></script>
+  <meta charset="utf-8">
+
+  <body>
+    <p>Please check the Javascript console for program output.</p>
+    </body>
+</html>
```

Comment:
This is almost right (I'm assuming it worked, but when we get to writing more HTML you'll have issues). It should look like: 

```
<html>
  <head>
    <title>Javascript Excercises</title>
    <script src="js/homework_02.js"></script>
    <meta charset="utf-8">
  </head>

  <body>
    <p>Please check the Javascript console for program output.</p>
  </body>
</html>
```